CMakeLists.txt

```diff
@@ -120,7 +126,14 @@ add_library(vcpkglib
     ${VCPKGLIB_BASE_INCLUDES}
     ${VCPKGLIB_INCLUDES}
 )
-target_include_directories(vcpkglib PUBLIC include)
+target_include_directories(vcpkglib PUBLIC
+    $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>
+    $<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}>
+)
+if (UNIX)
+    set_property(TARGET vcpkglib PROPERTY OUTPUT_NAME vcpkg)
```

Comment:
I'd prefer if we did something like forcing static linking in this case; `vcpkglib` is _not_ supposed to be shipped, it should just be a part of the vcpkg binary.

Reply:
Than it must be marked as `STATIC` and internal. In current behavior it will be installed, because it doesn't marked explicitly as static.